homeassistant/components/luftdaten/__init__.py

```diff
@@ -40,6 +40,12 @@
 SENSOR_PRESSURE_AT_SEALEVEL = "pressure_at_sealevel"
 SENSOR_TEMPERATURE = "temperature"
 
+SENSOR_LAEQ = "noise_LAeq"
+SENSOR_LA_MIN = "noise_LA_min"
+SENSOR_LA_MAX = "noise_LA_max"
```

Comment:
This is OK as Python module is getting all values and those are the names of the keys.

See https://github.com/home-assistant/core/pull/44548/files/5c90ed16321c695eb3f5a2586e669b9994139ebd#diff-4169c9621f3234cc26bb9b014c9f47648e3b0fc9dc1d752b0a8047421d4dfc36